docs/en/plugins/server.md

```diff
@@ -61,4 +61,59 @@ server:
   #
   # Default: 60s
   relay_timeout: 60s
-```
\ No newline at end of file
+```
+
+### Server initialization
+
+The `on_init` section is used for application initialization or warming up before starting workers. It allows you to set
+a command script that will be executed before starting the workers. You can also set environment variables to pass to
+this script.
+
+### Worker starting command
+
+The `server.command` option is required and is used to start the worker pool for each configured section in the config.
+
+> **Note:**
+> This option can be overridden by plugins with a pool section, such as the http.pool.command.
+
+The `user` and `group` options allow you to set the user and group that will start and own the worker process. An empty
```

Comment:
We should also mention, that to use this functionalily, RR should be started from the root. All temporary files (http for example) would be created with the provided user/group. Root needed only to have the ability to fork process under the different user.